common/web/keyboard-processor/tests/dom/cases/domKeyboardLoader.spec.ts

```diff
@@ -15,8 +15,8 @@ const device: DeviceSpec = {
 
 describe('Keyboard loading in DOM', function() {
   afterEach(() => {
-    if(window['KeymanWeb']) {
-      window['KeymanWeb'].uninstall();
+    if(window['KeymanWeb' as any]) {
+      (window['KeymanWeb' as any] as any).uninstall();
```

Comment:
Try using this at the file's top:

```typescript
declare let KeymanWeb: KeyboardInterface
```

It's possible to declare globals with type, after which you won't need the `as any` stuff.

We already use this pattern in a few places, such as the spot below:

https://github.com/keymanapp/keyman/blob/b15dcef5262df8e92851cdc62a7c00b910755b16/web/src/app/ui/kmwuitoggle.ts#L9

With that in place, it doesn't fix the `window['keyman']` pattern, but it does allow us to just assume that `keyman` exists as a global, which achieves what we're really after anyway.

Reply:
This didn't work exactly like this, but I found a similar pattern in `test_utils.ts`, so I applied that.

Reply:
I've submitted a patch PR as #11993 to implement the pattern I suggested.